test/travis_ci/docker/ubuntu_18_04/install_ubuntu.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -v
+
+# setup repo with recent gcc versions
+add-apt-repository -y ppa:ubuntu-toolchain-r/test
```

Comment:
this is cruft from 14.04. there should be no need to use ppa's with 18.04 as the default package versions should be recent enough. can you please use apt-get install for these packages now?